### PR TITLE
FE-857 Cypress Integrations tests for Alerts Create/Edit Pages

### PR DESCRIPTION
**Pull request by kchu1337**
### What Changed
 - Updated Alert form with data-id

### How To Test
 - npm run test-integration
 - npm run test-integration-headless

### To Do
- [ ] Add any additional tests

### CODE REVIEW

File: cypress/tests/integration/alerts/createPage.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    cy.get('[data-id="alert-filters"]').within(() => {
+      cy.findByLabelText('Subaccounts').type('subaccount');
+      cy.findByText('Master and all subaccounts').should('be.visible');
+
+      cy.findByText('Fake Subaccount 1 (101)').click({ force: true });
+      cy.findByText('Master and all subaccounts').should('not.be.visible');
+      cy.findByText('Master account').should('be.visible');
+    });
+  });
```
Comment (nicklemmon) [inline cypress/tests/integration/alerts/createPage.js]: It's probably worth adding a test actually submitting the form - and obviously stubbing the response.

File: cypress/tests/integration/alerts/editPage.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  it('On health score metric, changing filter facet resets the filter values', () => {
+    cy.get('[data-id="alert-filters"]').within(() => {
+      cy.findByLabelText('Facet').should('have.value', 'mailbox_provider');
+      cy.findByText('Gmail').should('be.visible');
+      cy.findByText('Apple').should('be.visible');
+      cy.findByLabelText('Facet').select('ip_pool');
+      cy.findByText('Gmail').should('not.be.visible');
+      cy.findByText('Apple').should('not.be.visible');
+    });
```
Comment (nicklemmon) [inline cypress/tests/integration/alerts/editPage.js]: Same here - handling submitting the form, what happens when the response is a `200` and what happens when you get a `400` are definitely worth testing.

File: cypress/tests/integration/alerts/createPage.js (hunk 1 above)
Comment (kchu1337) [reply cypress/tests/integration/alerts/createPage.js]: Yeah, I started your review and noticed that I was missing those.

File: cypress/tests/integration/alerts/createPage.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+      cy.findByText('Required').should('be.visible');
+      cy.findByText('At least one notification channel must be not empty').should('be.visible');
+    });
+
+    it('should show error message when alert creation fails', () => {
+      cy.stubRequest({
+        method: 'POST',
+        statusCode: 403,
+        url: '/api/v1/alerts',
+        fixture: 'alerts/400.post.json',
```
Comment (nicklemmon) [inline cypress/tests/integration/alerts/createPage.js]: Ideally the file name and the status code would line up just for consistency's sake. I've been using `400` everywhere

File: cypress/tests/integration/alerts/editPage.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  });
+
+  it.only('handles alert update successfully and redirect to details afterwards', () => {
```
Comment (nicklemmon) [inline cypress/tests/integration/alerts/editPage.js]: Looks like an only was left in - I think ESLint should have caught this before committing?

**Pull request merged**